home/models.py

```diff
@@ -59,6 +58,7 @@ class RSSImport(models.Model):
     def __str__(self):
         return self.rss_title_en
 
+
```

Comment:
Best practice is to commit them in an extra commit with message `Gardening` or similar. You can do it before or after your changes.

We could enable linting on travis to catch this early on.